WebPortal/i18n/nrcan/initiatives.json

```diff
@@ -219,7 +219,10 @@
       "Reports": "Reports",
       "SecurityClassification": "Security Classification",
       "UpdateAssignment": "Update",
-      "IncludeInCatalog": "Include in catalog"
+      "IncludeInCatalog": "Include in catalog",
```

Comment:
Any particular reason we don't put the full english in the code and only the french translation in the french file?